akka-stream-tests/src/test/scala/akka/stream/scaladsl/StreamRefsSpec.scala

```diff
@@ -433,6 +433,41 @@ class StreamRefsSpec extends AkkaSpec(StreamRefsSpec.config()) {
       }
     }
 
+    "not die to a slow and eager subscriber" in {
```

Comment:
Would this consistently fail without the fix?

Reply:
Yes, checkout the first commit to see.